Fix some bugs

YanagiEiichi opened the pull request.

YanagiEiichi committed:
Fix "Uncaught ReferenceError: module is not defined"

YanagiEiichi committed:
Fix "Error: [jqLite:onargs]"

YanagiEiichi committed:
Update grunt-contrib-uglify

YanagiEiichi committed:
Prevent default at ontouchstart

YanagiEiichi committed:
Don't tap on disabled element

YanagiEiichi committed:
Update grunt-contrib-coffee

YanagiEiichi committed:
Fix UMD global case

YanagiEiichi committed:
Fix "$", "trigger", and "originalEvent" not found

lelandcope commented:
Please make these changes in the `src/lc-touch.coffee`. The files you made changes to are the distribution files which the src file is compiled from. Once the changes are made I will review it. Let me know if you have any questions.

Commit:
Send $event

Commit:
Remove ngTapOutside which is buggy and not commonly used

Commit:
Readme clean up

YanagiEiichi committed:
Merge pull request #1 from CarterLi/master

Send $event as other ngEvents does. Remove ngTapOutside which is buggy and not very useful

CarterLi committed:
Grunt file cleanup

CarterLi committed:
Fix swipeLeft & swipeUp; Toggle ng-tap-active class like ngClick does; Add a simple html page for testing

CarterLi committed:
Remove the old example
It is tooooooo old ( embeds angular 1.1.5 Oo ) and may not work with recent changes

CarterLi committed:
Merge pull request #2 from CarterLi/master

Some changes

Commit:
Add ng-hold event

YanagiEiichi committed:
Merge pull request #3 from CarterLi/master

Add ng-hold event
